client/src/components/QandA/QuestionsList/Answers/AnswerImages.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const AnswerImages = ({ photos }) => {
+  return (
+    <div>
+      {photos.map(image =>
+        <img id='Image-Answer' src={image} />
```

Comment:
give each img tag a key attribute

Reply:
Handled. However, there still is one pesky component demanding a key prop and I feel pretty certain I've provided it with a key that is different. Don't entirely understand what's up with that one.